src/js/control-bar/skip-buttons/skip-forward.js

```diff
@@ -0,0 +1,66 @@
+import Button from '../../button';
+import Component from '../../component';
+
+/**
+ * Button to skip forward a configurable amount of time
+ * through a video. Renders in the control bar.
+ *
+ * e.g. options: {controlBar: {skipButtons: forward: 5}}
+ *
+ * @extends Button
+ */
+class SkipForward extends Button {
+  constructor(player, options) {
+    super(player, options);
+
+    this.validOptions = [5, 10, 30];
+    this.skipTime = this.getSkipForwardTime();
+
+    if (this.skipTime && this.validOptions.includes(this.skipTime)) {
+      this.controlText(`Skip forward ${this.skipTime} seconds`);
```

Comment:
Same comment here about localization.

Reply:
I've now moved the control text strings from `skip-forward` and `skip-backward` to `en.json` for localization.